k8s/staging/deployment.yaml

```diff
@@ -41,25 +41,30 @@ spec:
               value: staging
             - name: LOCAL_URL
               value: http://arcgis-proxy.gfw.svc.cluster.local:5700
-            - name: CT_URL
+            - name: GATEWAY_URL
               valueFrom:
                 secretKeyRef:
-                  key: CT_URL
+                  key: GATEWAY_URL
                   name: mssecrets
-            - name: CT_TOKEN
+            - name: MICROSERVICE_TOKEN
               valueFrom:
                 secretKeyRef:
-                  key: CT_TOKEN
+                  key: MICROSERVICE_TOKEN
                   name: mssecrets
-            - name: CT_REGISTER_MODE
+            - name: API_VERSION
```

Comment:
API_VERSION should be removed, it's no longer used. Not sure if this is also present on other PRs and I just missed it, but if I did, please go back and remove it